custom/ilsgateway/tests/test_report_runner.py

```diff
@@ -100,6 +118,15 @@ def test_report_runner(self):
             )
 
             self._move_location(self.facility1, self.district2.get_id)
+
+            pr = PendingReportingDataRecalculation.objects.filter(sql_location=self.facility1.sql_location).first()
+            self.assertEqual(pr.type, 'parent_change')
+            self.assertDictEqual(
+                pr.data,
+                {'previous_parent': self.district1.get_id, 'current_parent': self.district2.get_id}
+            )
+
+            report_run(TEST_DOMAIN)
```

Comment:
I'm not sure that I understand. Below you can see assertions for respective location's ancestors. ```report_run``` is recalculating data for all locations from TEST_DOMAIN.